src/Payments.Core/Domain/FinancialAccount.cs

```diff
@@ -24,9 +24,20 @@ public class FinancialAccount
         [Required]
         public string Account { get; set; }
 
+        [StringLength(4)]
+        [Required]
+        public string Object { get; set; }
+
         [StringLength(5)]
         public string SubAccount { get; set; }
 
+        [StringLength(3)]
+        [DisplayFormat(NullDisplayText = "---")]
+        public string SubObject { get; set; }
+
+        [StringLength(9)]
+        public string Project { get; set; }

```

Comment:
@jSylvestre add null display text to project and subacct too?

Reply:
@srkirkland This code is currently commented out. I'll just delete it.